src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/view-your-data/opentelemetry-logs-page.mdx

```diff
@@ -9,9 +9,9 @@ metaDescription: Here are some tips for understanding the OpenTelemetry logs pag
 
 After you've sent us your OpenTelemetry data and opened your service (entity) in the UI, you can click on **Logs** to see data about your application. For more information about how to associate log data to your application in New Relic, see our [OpenTelemetry and logging documentation](/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/best-practices/opentelemetry-best-practices-logs).
 
-For your data to appear in this section, make sure it has the following:
+For your data to appear in this section, do the following:
```

Comment:
"this section" feels odd to me. How about "this section of the UI"?

Reply:
@paperclypse I made this change.